cluster-cm/src/test/java/com/sequenceiq/cloudbreak/cm/ClouderaManagerClusterHealthServiceTest.java

```diff
@@ -229,6 +255,31 @@ void testServicesNotRunningOnHosts() throws ApiException {
         assertThat(result.areServicesNotRunning(hostName("host-3"))).isFalse();
     }
 
+    @Test
```

Comment:
We should another test case(s) to check that getDisconnectedNodeManagers should return empty list.